src/content/pages/forms.md

```diff
@@ -0,0 +1,14 @@
+---
+title: "Forms"
+description: "this is meta description"
+draft: false
+---
+
+#### [Reimbursement Form](https://form.jotform.com/Troop150/troop-150-reimbursement-form)
```

Comment:
Let's figure out if there's a way to be more clear on what is a link (vs a section header).